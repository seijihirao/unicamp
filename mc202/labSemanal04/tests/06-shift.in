is-empty
unshift 10   Quanto melhor é quando há bruma.
unshift 9    A distinção entre nada e coisa nenhuma.
unshift 8    Estudar é uma coisa em que está indistinta
unshift 7    Livros são papéis pintados com tinta.
unshift 6    E a brisa, essa, de tão naturalmente matinal, como tem tempo, não tem pressa...
unshift 5    O rio corre bem ou mal, sem edição original.
unshift 4    O sol doira sem literatura. 
unshift 3    Ler é maçada, estudar é nada.
unshift 2    Ter um livro para ler - e não o fazer!
unshift 1    Ai que prazer - não cumprir um dever.
is-empty
print-first
shift
print-first
shift
print-first
shift
print-first
shift
print-first
shift
print-first
shift
print-first
shift
print-first
shift
print-first
shift
print-first
shift
print-first
shift
print-first
shift
print-first
is-empty
exit
