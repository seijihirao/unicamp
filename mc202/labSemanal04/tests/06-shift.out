yep
nope
1    Ai que prazer - não cumprir um dever.
2    Ter um livro para ler - e não o fazer!
3    Ler é maçada, estudar é nada.
4    O sol doira sem literatura.
5    O rio corre bem ou mal, sem edição original.
6    E a brisa, essa, de tão naturalmente matinal, como tem tempo, não tem pressa...
7    Livros são papéis pintados com tinta.
8    Estudar é uma coisa em que está indistinta
9    A distinção entre nada e coisa nenhuma.
10   Quanto melhor é quando há bruma.
yep
